### PR TITLE
manifest: update nrfxlib

### PR DESCRIPTION
**Pull request by lemrey**
Update nrfxlib to include bsdlib 0.5.0.
The main change is the removal of `bsd_irrecoverable_error_handler()`.

### CODE REVIEW

**Comment by lemrey** [comment]
Updated `nrfxlib` SHA: bf7173d86ebeeee5ea87b51995a00c44da3ea667.
Added RAW socket offloading.

File: samples/nrf9160/http_application_update/src/main.c [hunk 1]
```diff
@@ -190,7 +182,10 @@ void main(void)
 		printk("Modem firmware update failed\n");
 		printk("Fatal error.\n");
 		break;
-
+	case -1:
+		printk("Could initialize bsdlib.\n");
```
Comment (rlubos) [inline samples/nrf9160/http_application_update/src/main.c]: `Could not`?

Comment (lemrey) [reply samples/nrf9160/http_application_update/src/main.c]: Thanks.

**Comment by rlubos** [comment]
It seems that you've received manifest update for free in #1333, we should merge the remaining fixes asap.

**Comment by rlubos** [comment]
I took the convenience to rebase your branch and fix the conflict.

**Pull request merged**